tests/test_memory_core/test_memory_core_genesis2.py

```diff
@@ -85,7 +85,7 @@ def read_and_write(self, addr, data):
         self.eval()
 
 
-def test_sram_basic():
+def sram_basic():
```

Comment:
Why remote `test_`?

Reply:
this was to prevent it from running this test, but I'm removing this file - unused